packages/tooltip/src/types.ts

```diff
@@ -12,6 +12,11 @@ export interface IUseTooltipProps<T = HTMLElement> {
   delayMilliseconds?: number;
   /** Specifies the tooltip ID */
   id?: string;
+  /**
+   * Indicates that the tooltip is used as the trigger's primary label rather
+   * than as a supplemental description
+   */
```

Comment:
[nit] For clarity, should we provide usage recommendations?


```suggestion
  /**
   * Indicates that the tooltip is used as the trigger's primary label rather
   * than as a supplemental description.
   * Use this for icon-only buttons that lack a visible text label. 
   */
```

Reply:
I'm not confident we know this added guidance to be true. There could be valid cases for applying `isLabel` to other types of triggers.